### ISSUE TITLE
crashing with a game open.

### ISSUE THREAD

**Issue by Miyazono1** [HookedBehemoth/ShareNX-Overlay#6]
as the title says. if i go to use the overlay with a game open, it crashes back to hekate. this only seems to happen when there is no screenshot to upload, so it's not a huge issue, but it is there.

**Comment by HookedBehemoth** [comment]
Does this happen on the latest version?

**Comment by Miyazono1** [comment]
yeah. if you open it with no screenshot in the home menu it's fine, but if you do it in game it crashes. i was playing xenoblade chronicles 2. i haven't tested in other games to see if it happens there as well, but yes it is the latest version.
